application-analytics-ui/src/main/resources/Analytics/WebHome.xml

```diff
@@ -27,45 +27,187 @@
   <defaultLanguage>en</defaultLanguage>
   <translation>0</translation>
   <creator>xwiki:XWiki.Admin</creator>
-  <parent>Main.WebHome</parent>
+  <parent>xwiki:Main.WebHome</parent>
   <author>xwiki:XWiki.Admin</author>
   <contentAuthor>xwiki:XWiki.Admin</contentAuthor>
   <version>1.1</version>
   <title>Analytics</title>
   <comment/>
   <minorEdit>false</minorEdit>
-  <syntaxId>xwiki/2.1</syntaxId>
+  <syntaxId>xwiki/2.0</syntaxId>
```

Comment:
Why is this needed?

Reply:
When I created the new homepage using the dashboard template, it also changed this property.

Reply:
Can you check if there are problems with using `2.1`?